src/selection-sharer.js

```diff
@@ -232,6 +232,7 @@
     this.shareFacebook = function(e) {
       e.preventDefault();
       var text = self.htmlSelection.replace(/<p[^>]*>/ig,'\n').replace(/<\/p>|  /ig,'').trim();
+      text = text.replace(/<\/?[^>]+(>|$)/g, ""); // Remove HTML tags, like <h3>foo</h3>bar => foobar
```

Comment:
Nice catch. However, I'm not sure why `self.htmlSelection` was chosen here. I think `self.textSelection` would make a lot more sense as it already gives us just plain text. @xdamman, would you like to chime in here?